src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -0,0 +1,651 @@
+
+    existing_notices: ResultProxy = bind.execute(text(notice_query))
+    for existing_notice in existing_notices:
+        notice_key = existing_notice["notice_key"]
+        component_mapping = NOTICES_TO_EXPERIENCE_CONFIG.get(notice_key)
+        if component_mapping:
+            regions_to_add = set(existing_notice["regions"]).difference(
+                REGIONS_TO_REMOVE
+            )
+            if existing_notice["displayed_in_overlay"]:
+                experience_config = component_mapping.get(ComponentType.Overlay)
+                reconciled_experience_config_map[experience_config]["regions"].update(
+                    regions_to_add
+                )
+                reconciled_experience_config_map[experience_config][
+                    "privacy_notices"
+                ].add(existing_notice["id"])
+                reconciled_experience_config_map[experience_config][
+                    "needs_migration"
+                ] = True
+
+            if existing_notice["displayed_in_privacy_center"]:
+                experience_config = component_mapping.get(ComponentType.PrivacyCenter)
+                reconciled_experience_config_map[experience_config]["regions"].update(
+                    regions_to_add
+                )
+                reconciled_experience_config_map[experience_config][
+                    "privacy_notices"
+                ].add(existing_notice["id"])
+                reconciled_experience_config_map[experience_config][
+                    "needs_migration"
+                ] = True
+
+    existing_experience_configs: ResultProxy = bind.execute(
+        text(experience_config_query)
+    )
+    has_existing_experience_configs = False
+    for eec in existing_experience_configs:
+        has_existing_experience_configs = True
+        component = ComponentType(eec["component"])
+        configs = []
+
+        # because we've given experiences a location dimension in this migration,
+        # an existing experience effectively maps to two "new" default experiences -
+        # one that is surfaced in the US (and is associated with "US" notices), and
+        # one that is surfaced in the EU (and is associated with the "EU" notices).
+        # because of this, we'll copy over existing experience text/attributes to _both_
+        # of these corresponding new experience configs.
+        if eea_config_id := EEA_MAPPING.get(component):
+            configs.append(reconciled_experience_config_map.get(eea_config_id))
+        if us_config_id := US_MAPPING.get(component):
+            configs.append(reconciled_experience_config_map.get(us_config_id))
+        for config in configs:
+            if config:
+                config["accept_button_label"] = eec["accept_button_label"]
```

Comment:
Have we considered automatically enabling/disabling based on their underlying data?

Reply:
yeah, so is this then something we should mark as a manual followup?

Reply:
yes I think that would make sense

Reply:
noted "Determine whether any changes are desired for the accept_button_label on experience configs" as a bullet point in the manual followups

Reply:
I wasn't talking about the `accept_button_label` specifically here I was just talking about adding `disabling` here as a field we bring over in addition

Reply:
ohhh sheesh that makes more sense now 😅 

i'd already had that as a followup, so we should be good 👍 
> Enable appropriate experience configs - all will be disabled by default!